canonical-web-and-design/ubuntu.com
# Demo of /advantage using contracts.staging.canonical.com

**DO NOT RELEASE**

## Done

Demo of /advantage using contracts.staging.canonical.com

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Sign in and see data from the staging environment
